"exo compute instance create" with --private-network throws panic

Hi,
me again... still migrating to the new exo cli compute command...

"exo compute instance create" with --private-network throws following error:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x7ad996]
```

Example:

```
$ exo version
exo 1.45.1 652912ab (egoscale 0.80.1)

$ exo compute private-network show --zone at-vie-1 molecule-test-private-network
┼─────────────────┼──────────────────────────────────────┼
│ PRIVATE NETWORK │                                      │
┼─────────────────┼──────────────────────────────────────┼
│ ID              │ 33fa8fe6-bcbb-4b23-8a8f-2c7c4531698b │
│ Name            │ molecule-test-private-network        │
│ Description     │                                      │
│ Zone            │ at-vie-1                             │
│ Type            │ manual                               │
┼─────────────────┼──────────────────────────────────────┼

// Command works without the --private-network
$ exo compute instance create --output-format json --disk-size "10" --template "Linux Ubuntu 20.04 LTS 64-bit" --zone "at-vie-1" --instance-type "standard.micro" "molecule-test"
 ✔ Creating instance "molecule-test"... 21s
{"id":"c0f2d04d-6a62-44da-8058-84a1527df37f","name":"molecule-test","creation_date":"2021-10-20 12:54:12 +0000 UTC","instance_type":"standard.micro","template_id":"Linux Ubuntu 20.04 LTS 64-bit","zoneid":"at-vie-1","anti_affinity_groups":[],"security_groups":["default"],"private_networks":[],"elastic_ips":[],"ip_address":"185.150.X.X","ipv6_address":"-","ssh_key":"-","disk_size":"10 GiB","state":"running","labels":null}

// Command does not work with --private-network
$ exo compute instance create --output-format json --disk-size "10" --template "Linux Ubuntu 20.04 LTS 64-bit" --zone "at-vie-1" --instance-type "standard.micro" --private-network "molecule-test-private-network" "molecule-test-2"
 ⠙ Creating instance "molecule-test-2"... 21s
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x7ad996]

goroutine 28 [running]:
github.com/exoscale/egoscale/v2.(*Client).AttachInstanceToPrivateNetwork(0xc0003fe420, {0x17240c0, 0xc0003e89c0}, {0x7fff60921253, 0x8}, 0xc0000c06e0, 0xc0006094f0, {0xc0006d3f68, 0x1, 0x1})
        github.com/exoscale/egoscale@v0.80.1/v2/instance.go:252 +0xf6
github.com/exoscale/cli/cmd.(*instanceCreateCmd).cmdRun.func4()
        github.com/exoscale/cli/cmd/instance_create.go:211 +0x198
github.com/exoscale/cli/cmd.decorateAsyncOperation.func2(0x0)
        github.com/exoscale/cli/cmd/output.go:316 +0x22
created by github.com/exoscale/cli/cmd.decorateAsyncOperation
        github.com/exoscale/cli/cmd/output.go:315 +0x3de
```

The instance gets created but will not be in the private network.

Any ideas?

--- a/cmd/instance_create.go
+++ b/cmd/instance_create.go
@@ -208,7 +208,7 @@ func (c *instanceCreateCmd) cmdRun(_ *cobra.Command, _ []string) error {
 		}
 
 		for _, p := range privateNetworks {
-			if err = cs.AttachInstanceToPrivateNetwork(ctx, c.Zone, instance, p, nil); err != nil {
+			if err = cs.AttachInstanceToPrivateNetwork(ctx, c.Zone, instance, p); err != nil {
 				return
 			}
 		}

